### DISCUSSION TITLE
Adding another Tesla

### DISCUSSION THREAD

**Discussion by dettofatto** [llamafilm/tesla-http-proxy-addon#156] [Q&A] [answered]
Hy! I've added my Tesla and now I would like add another Tesla with another account.
It's possible?
I'm using 
HA supervised with this addon.
Thank you

**Comment by llamafilm** [comment, marked as answer]
The proxy addon should support this, but the Tesla integration does not. I actually tried making this work and got stumped by the `configflow` logic.

**Comment by dettofatto** [comment]
So, for another car with a different account what do you suggest? Another instance of ha on another raspberry? Or another docker (but the network configuration may be hard)?

**Reply by Lich2023** [reply]
Any new ideas to Work with two teslas in two Accounts?

**Reply by dettofatto** [reply]
Two hardware...
